miden-lib/asm/kernels/transaction/lib/account.masm

```diff
@@ -341,6 +344,81 @@ export.apply_storage_offset
     # => [offset_slot_index]
 end
 
+#! Validates all account procedures storage offsets
```

Comment:
nit: As someone who didn't have too much context as to how this works, maybe this could explain what validating the offsets means (ie, being within the correct storage slots bounds)? Since it's not immediately clear.